### PR TITLE
WEBUI-557: make create version button usable with keyboard

### PR DESCRIPTION
**Pull request by rakeshkumar1019**

### CODE REVIEW

**Review by swarnadipa-dev — DISMISSED** [review]
looks good

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by GuruprasadAnk1996 — DISMISSED** [review]

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks good

File: elements/nuxeo-document-versions/nuxeo-document-create-version.js [hunk 1]
```diff
@@ -64,6 +64,23 @@ Polymer({
...
+        background-color: transparent;
+        color: inherit;
+        font-family: inherit;
+        font-size: inherit;
+        cursor: pointer;
+        text-align: center;
+        transition: background 250ms ease-in-out, transform 150ms ease;
+        -webkit-appearance: none;
+        -moz-appearance: none;
+      }
```
Comment (semisse) [inline elements/nuxeo-document-versions/nuxeo-document-create-version.js]: Testing it on Chrome, Safari and Firefox I concluded that we don't need all these styles. Please consider the following change:
```suggestion
      #createButton {
        margin: 0;
        padding: 0;
        background-color: transparent;
        border: none;
      }
```

Comment (rakeshkumar1019) [reply elements/nuxeo-document-versions/nuxeo-document-create-version.js]: Thankyou @semisse, made the changes.

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good.

**Review by Gabez0r — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**